R/write_activity_tsv.R

```diff
@@ -24,7 +24,7 @@ write_activity_tsv <- function(output_dir, input_csv, compound_tsv, assay_tsv,
 
   #<- input_activity$now[is.na(input_activity$Sample)]
   # Pre-allocate with NA
-  input_activity$matched_AIDX <- NA
+  #input_activity$matched_AIDX <- NA
```

Comment:
I don't think you need to preallocate this to `NA`, as your `sapply` later will always return either `NA` or a string.